### PR TITLE
Issue 4533 m

### PR DESCRIPTION
**Pull request by rmroot**
connects #4533 

updates application to Angular 13 and other latest packages.
Removes older plotly dependency and uses angular service
Removes other unused code/components and dependencies

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -99,8 +110,12 @@
           "configurations": {
             "production": {
               "browserTarget": "amo-tools-desktop:build:production"
+            },
+            "development": {
+              "browserTarget": "verifi:build:development"
```
Comment (nbintertech) [inline angular.json:115]: verifi should say 'amo-tools-desktop'

File: package.json [hunk 2]
```diff
@@ -8,11 +8,11 @@
...
   "angular-cli": {},
   "engines": {
-    "node": "12.8.0",
-    "npm": "6.10.2"
+    "node": "14.15.0",
+    "npm": "6.14.8"
   },
   "scripts": {
-    "ng-high-memory": "node --max_old_space_size=8000 ./node_modules/@angular/cli/bin/ng build --prod --aot",
+    "ng-high-memory": "node --max_old_space_size=8000 ./node_modules/@angular/cli/bin/ng build --configuration production --aot",
```
Comment (nbintertech) [inline package.json:15]: I think it's '--configuration=production'

File: angular.json (hunk 1 above)
Comment (rmroot) [reply angular.json]: fixed

File: src/app/calculator/system-and-equipment-curve/system-and-equipment-curve-graph/system-and-equipment-curve-graph.component.ts [hunk 3]
```diff
@@ -211,41 +188,69 @@ export class SystemAndEquipmentCurveGraphComponent implements OnInit {
...
+      traceData.push(trace);
     });
+    this.dataPointTraces.forEach(trace => {
+      traceData.push(trace);
+    })
+    if (this.expanded && this.expandedSystemChartDiv) {
+      this.plotlyService.newPlot(this.expandedSystemChartDiv.nativeElement, traceData, chartLayout, this.curveEquipmentChart.config)
+        .then(chart => {
+          chart.on('plotly_click', (graphData) => {
+            this.createDataPoint(graphData);
```
Comment (nbintertech) [inline src/app/calculator/system-and-equipment-curve/system-and-equipment-curve-graph/system-and-equipment-curve-graph.component.ts:213]: No hover data showing in in table when expanded. Looks like the event listened and method just need to be added for the expanded block

**Review by nbintertech — CHANGES_REQUESTED** [review]

File: src/app/calculator/system-and-equipment-curve/system-and-equipment-curve-graph/system-and-equipment-curve-graph.component.ts (hunk 3 above)
Comment (rmroot) [reply src/app/calculator/system-and-equipment-curve/system-and-equipment-curve-graph/system-and-equipment-curve-graph.component.ts:213]: fixed

**Review by nbintertech — APPROVED** [review]

**Pull request merged**